packages/client/src/hooks/authentication.hooks.ts

```diff
@@ -109,7 +109,7 @@ export function useSignUpWithEmailAndPassword(navigate: NavigateFunction, ctx?:
  * Hook - Authentication
  * Login user with email and password
  */
-export function useLoginWithEmailAndPassword(navigate: NavigateFunction, ctx?: IHookContext): IHookLoginWithEmailAndPassword {
+export function (navigate: NavigateFunction, ctx?: IHookContext): IHookLoginWithEmailAndPassword {
```

Comment:
Where is function name ?

Reply:
You should not touch this function, you should take inspiration to build your hook

Reply:
Sorry , happened by mistake